docs/helm-chart/production-guide.rst

```diff
@@ -332,6 +332,7 @@ In the Airflow Helm chart, the ``securityContext`` can be configured in several
   * :ref:`uid <parameters:Airflow>` (configures the global uid or RunAsUser)
   * :ref:`gid <parameters:Airflow>` (configures the global gid or fsGroup)
   * :ref:`securityContext <parameters:Kubernetes>` (same as ``uid`` but allows for setting all `Pod securityContext options <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#podsecuritycontext-v1-core>`_)
+  * :ref:`securityContexts <parameters:Kubernetes>` (same as ``securityContext`` with additional security context on the container level `Container securityContext options <https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#securitycontext-v1-core>`_)
 
 The same way one can configure the global :ref:`securityContext <parameters:Kubernetes>`, it is also possible to configure different values for specific workloads by setting their local ``securityContext`` as follows:
```

Comment:
Since we are deprecating `securityContext`, we need to update this example to show `securityContexts` instead.